templates/404.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% block title %}404 - Cobwebs{% endblock %}
+{% block content %}
+  <div id="center">
+    <h1 class="red">404 - Cobwebs</h1>
+    <h2>Nothing is here...</h2>
+    <h2><a href="/">Try again from the start</a></h2>
```

Comment:
Would it be a good idea for us to re-style links so they fit in better? It doesn't look the greatest in the screenshot.

Reply:
the users won't be on the page for long, and shouldn't even be there in the first place, therefore its more of a UX thing for it to be left the default color rather than fitting in with the page itself

Reply:
Alright, fair